repositories/repositories.bzl

```diff
@@ -124,6 +124,24 @@ def ros2_repositories():
         urls = ["https://github.com/facebook/zstd/releases/download/v1.5.2/zstd-1.5.2.tar.gz"],
     )
 
+    maybe(
+        http_archive,
+        name = "lz4",
+        build_file = "@com_github_mvukov_rules_ros2//repositories:lz4.BUILD.bazel",
+        sha256 = "4ec935d99aa4950eadfefbd49c9fad863185ac24c32001162c44a683ef61b580",
+        strip_prefix = "lz4-1.9.3",
```

Comment:
Is this version referenced by mcap version below?

Reply:
Yes, mcap references this specific version.